### PR TITLE
remove quantmaxpool

### PR DESCRIPTION
**Pull request by capnramses**
### Purpose of this PR

* Remove unused `quantmaxpool` functionality. Issue #744.
* Familiarising myself with the dev & PR process by taking a small task.

### Changes made in this PR

* Removed `quant_max_pool` functionality within files.
* Deleted 2x `pool.py` files that only used `quant_max_pool` functions.
* Removed import statements referring to the above files and functions.
* Removed inclusion of the above in handler lists.

### Testing Summary

- [x] Ran all the `brevitas_examples` tests before and after these edits.
- [x] Ran a full local `nox` pass over the tests.
- [x] Tested this Brevitas branch in a typical quantisation program of my own.

### Risk Introduced / Breaking Changes

* I'm not 100% familiar with how this functionality was used, so I may have some testing gaps.

### Pre-Merge Checklist

- [x] Refresh branch from dev, resolve conflicts.
- [x] Reviewed own PR and checked code.
- [x] Initial PR CI/CD passing.
- [x] Reviewed and approved.
- [x] Full CI/CD passing.

### CODE REVIEW

File: src/brevitas/export/onnx/standard/qoperator/manager.py
Comment (capnramses) [inline src/brevitas/export/onnx/standard/qoperator/manager.py:1]: precheck failing here and in the other manager.py file.

**Comment by Giuseppe5** [comment]
We always merge PRs to `dev` and not `master` :)

File: src/brevitas/export/onnx/standard/qoperator/manager.py
Comment (capnramses) [reply src/brevitas/export/onnx/standard/qoperator/manager.py:1]: fixed by running `pre-commit run --all-files`. it was just a whitespace/formatting check failing (needed the double linebreak before class).

File: src/brevitas/export/onnx/standard/qoperator/manager.py [hunk 1]
```diff
@@ -53,9 +51,7 @@ class StdQOpONNXManager(StdONNXBaseManager):
         StdQOpONNXQuantHardTanhHandler,
         StdQOpONNXQuantIdentityHandler,
         StdQOpONNXQuantTanhHandler,
-        StdQOpONNXQuantSigmoidHandler,
-        StdQOpONNXQuantMaxPool1d,
-        StdQOpONNXQuantMaxPool2d]
+        StdQOpONNXQuantSigmoidHandler,]
```
Comment (Giuseppe5) [inline src/brevitas/export/onnx/standard/qoperator/manager.py:54]: Remove comma

**Review by Giuseppe5 — APPROVED** [review]

**Pull request merged**